fix: Change "Single Account" to "Single Project" in button text

samantha-t28 opened the pull request:
## Summary

This pull request resolves [#198879](https://github.com/elastic/kibana/issues/198879), where "Single Account" was incorrectly displayed in the GCP integration screen. The text has been updated to "Single Project" to align with GCP terminology. Additionally, the paragraph above this text was updated to ensure consistency across the UI.

## Motivation

Aligning the GCP integration screen with GCP’s correct terminology by replacing "Single Account" with "Single Project" ensures clarity and consistency across the UI.

## Screenshots:
**Before**: interface showing "Single Account"

<img width="858" alt="Screenshot 2024-11-18 at 1 29 18 PM" src="https://github.com/user-attachments/assets/c03cb5f0-8e30-4e90-852d-aa2e91d6918d">

**After**: updated interface showing "Single Project"

<img width="782" alt="Screenshot 2024-11-18 at 1 29 57 PM" src="https://github.com/user-attachments/assets/2d1b3e09-8da8-401a-b711-34934c31a1b2">


### Checklist

Check the PR satisfies following conditions. 

Reviewers should verify this PR satisfies this list as well.

- [x] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/README.md)
- [ ] ~~[Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials~~
- [ ] ~~[Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios~~
- [ ] ~~If a plugin configuration key changed, check if it needs to be allowlisted in the cloud and added to the [docker list](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)~~
- [ ] ~~This was checked for breaking HTTP API changes, and any breaking changes have been approved by the breaking-change committee. The `release_note:breaking` label should be applied in these situations.~~
- [ ] ~~[Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed~~
- [x] The PR  description includes the appropriate Release Notes section, and the correct `release_node:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

<!--ONMERGE {"backportTargets":["8.x"]} ONMERGE-->

albertoblaz commented:
Hi @samantha-t28! Thanks a lot for contributing with your PR!

Could you please add a link to the [original issue](https://github.com/elastic/kibana/issues/198879) in the description?

I recommend you as well replacing the placeholder text ("Summarize your PR. If it involves visual changes include a screenshot or gif.") with a very brief summary that describes the change and motivation.

Also, I saw in the screenshots you changed the button's copy but the paragraph above still says "single account". Would you mind changing that as well to keep copy consistent across the UI?

<img width="742" alt="Screenshot 2024-11-18 at 12 00 22" src="https://github.com/user-attachments/assets/4fc1e6e7-4240-4e66-a114-742f69264cee">

samantha-t28 commented:
Hi @albertoblaz, I’ve added the original issue link to the description and included a brief summary and motivation for clarity. Additionally, I’ve updated the paragraph above the button from "Single Account" to "Single Project". Please let me know if there’s anything I might have missed or needs adjustment 🙂

maxcold reviewed (approved):
Looks good, thanks for the contribution!

albertoblaz reviewed (approved):
Thanks @samantha-t28!

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/252293)
* Commit: 53efc56e7bffcb315b77f25bc4735953f0429f48

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/252293","number":252293,"commit":"53efc56e7bffcb315b77f25bc4735953f0429f48"}],"number":252293}
buildkite-pr-comment-->

opauloh commented:
@elasticmachine run docs-build

kibanamachine commented:
Starting backport for target branches: 8.x

https://github.com/elastic/kibana/actions/runs/11920460198

kibanamachine commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.x|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/200813">](https://github.com/elastic/kibana/pull/200813)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

<!--- Backport version: 9.4.3 -->
